admin/controllers/class-webchangedetector-logs-controller.php

```
@@ -514,285 +518,314 @@ private function render_auto_update_history() {
+			<?php else : ?>
+				<?php foreach ( $update_history as $index => $entry ) : ?>
+					<div class="accordion-container" style="margin-top: 20px;">
+						<div class="accordion accordion-batch accordion-auto-update">
+							<div class="mm_accordion_title">
+								<h3>
+									<div style="display: inline-block;">
+										<div class="accordion-batch-title-tile accordion-batch-title-tile-status">
+											<?php
+											// Check if this is an error entry.
+											$is_error     = isset( $entry['error'] ) && ! empty( $entry['error'] );
+											$status_class = 'status-' . str_replace( '_', '', $entry['summary']['status'] );
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$status_class'. (undefined)

(UnusedLocalVariable)